ethminer/CMakeLists.txt

```diff
@@ -28,6 +28,32 @@ if (JSONRPC)
 	endif()
 endif()
 
+if (ETH_JSON_RPC_STUB)
```

Comment:
This is a cmake function. It's a custom command. Granted it has many parameters, but this is how jsonrpcstub works in our cmake files.